roles/node/tasks/install-vms.yml

```diff
@@ -10,3 +10,9 @@
   loop: "{{ avalanchego_vms_install }}"
   loop_control:
     loop_var: vm
+
+- name: Template aliases.json file for vms
+  template:
+    src: vm-aliases.json.j2
+    dest: "{{ avalanchego_vms_conf_dir }}/aliases.json"
+  notify: Restart avalanchego
```

Comment:
https://github.com/AshAvalanche/ansible-avalanche-collection/pull/109 changed the way VM binaries are symlinked to the node's `plugins` dir and removed the usage of aliases: a VM binary symlink now corresponds to the VM ID.

I think this is beneficial for 2 reasons:
1. This removes one layer of management/complexity (the VM aliases level)
2. This makes troubleshooting very easy (as the VM binary name is the VM ID)

I don't really see the benefit of reintroducing the VM aliases file: since Ansible should be the only tool used for managing the node, the VM will never be named after its alias (but after its ID).

Reply:
Makes sense and it simplifies things.